server/middleware/authenticator.js

```diff
@@ -30,11 +30,14 @@ class Authenticate {
     if (!req.headers.authorization) {
       return util.errorStatus(res, 403, 'Authentication is required');
     }
-
-    const token = req.headers.authorization.split(' ')[1];
+    let token = req.headers.authorization;
+    if(token.startsWith('bearer ')) {
```

Comment:
@encodedBicoding  please what is the reason for this line?

Reply:
The token is a string that starts with bearer, or doesn't. The way u implemented it before, it just splits the main token, buy that line checks first if there is bearer before the main token before splitting

Reply:
Well, I think that the if block is not necessary since the split token still gets verified in the try block that follows.

Reply:
I think I understand what @encodedBicoding is trying to do. He's making a provision to just insert the token without appending the word `Bearer`. I think it's a good idea to take away the stress of always appending that bearer word but I don't know if it's standard. Maybe I'll look it up after demo today.